src/Apps/Order/Components/__stories__/ShippingAndPaymentDetails.story.tsx

```diff
@@ -41,40 +59,64 @@ storiesOf("Apps/Order Page/Components", module).add(
         <Section title="Shipping and Payment Summary">
           <h4>Delivery</h4>
           <Flex flexDirection="column" width={300}>
-            <ShippingAndPaymentSummary order={order} />
+            <StorybookRelay
+              Component={ShippingAndPaymentSummary}
+              mockResolvers={{ Order: () => order }}
+              query={orderQuery}
+            />
```

Comment:
Nice 😎

Reply:
@sweir27 has been pushing for a solid way to test bits of Relay code for a while and this feels right